PINCE.py

```diff
@@ -94,13 +95,20 @@
     ("简体中文", "zh_CN")
 ]
 
+theme_list = ["Dark",
```

Comment:
I'd suggest moving this to the new theme setting file so theme related stuff are not spread out to different files.